packages/evolution-generator/src/scripts/generate_input_range.py

```diff
@@ -103,13 +104,13 @@ def generate_input_range(input_file: str, output_file: str):
             ts_code += f"{INDENT}maxValue: {max_value},\n"
             ts_code += f"{INDENT}formatLabel: (value, language) => {{\n"
             ts_code += f"{INDENT}{INDENT}return value < 0 ? '' : `${{value}} ${{language === 'fr' ? '{unit_fr}' : language === 'en' ? '{unit_en}' : ''}}`;\n"
-            # ts_code += f"{INDENT}{INDENT}return value + ' ' + (language === 'fr' ? '{unit_fr}' : '{unit_en}');\n"
-            ts_code += f"{INDENT}}}\n"
+            ts_code += f"{INDENT}}},\n"
+            ts_code += f"{INDENT}trackClassName: 'input-slider-{input_color}'\n"
             ts_code += "};\n\n"
 
-        # Write TypeScript code to a file
-        with open(output_file, mode="w", encoding="utf-8", newline="\n") as ts_file:
-            ts_file.write(ts_code)
+            # Write TypeScript code to a file
+            with open(output_file, mode="w", encoding="utf-8", newline="\n") as ts_file:
```

Comment:
Le bloc `with` a changé d'indentation ici. C'est voulu ou un accident?

Reply:
C'est mon auto formating qui a reperé l'erreur.

Reply:
Sauf que là ça va écrire plusieurs fois le code puisqu'il est initialisé avant la boucle for et que ça écrit à chaque boucle? Au moins ça écrase à chaque fois...

Reply:
Ta raison, pas besoin de réécrire à chaque fois. C'est corrigé.